src/utils/map-controls.js

```diff
@@ -86,17 +89,6 @@ export default class MapControls {
       srcEvent.ctrlKey || srcEvent.shiftKey);
   }
 
-  isRightButtonPressed(event) {
```

Comment:
This moved into mjolnir?

Reply:
Yes. https://github.com/uber-web/mjolnir.js/pull/5